packages/frameworks/src/frameworks.ts

```diff
@@ -1932,6 +1932,33 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Storybook',
+    slug: 'storybook',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/storybook.svg',
+    tagline: 'Frontend workshop for UI development',
+    description:
+      'Storybook is a frontend workshop for building UI components and pages in isolation.',
+    website: 'https://storybook.js.org/',
+    ignoreRuntimes: ['@vercel/next', '@vercel/node'],
+    useRuntime: { src: 'package.json', use: '@vercel/static-build' },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install`, `pnpm install`, or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build-storybook`',
```

Comment:
placeholder should have `or` (see others)

Reply:
Done!